server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/deleteOrphanTasks.groovy

```diff
@@ -0,0 +1,39 @@
+import org.apache.logging.log4j.LogManager
+import org.nuxeo.ecm.platform.task.Task
+import org.nuxeo.ecm.core.api.IdRef
+import org.nuxeo.ecm.core.api.ScrollResult
+import org.nuxeo.runtime.transaction.TransactionHelper
+import java.util.HashMap
+
+def log = LogManager.getLogger("taskGC.groovy")
+def nxql = "SELECT * FROM Document WHERE ecm:mixinType = 'Task'"
+def dryRun = true
+def session = Context.getCoreSession()
+def ScrollResult<String> scrollResult = session.scroll(nxql, 500, 60)
+def removedCount = 0
+def taskCount = 0
+
+log.info("Starting deleting orphan tasks...")
+while (scrollResult.hasResults()) {
+    taskCount += scrollResult.getResults().size()
+    cache = new HashMap<IdRef>()
```

Comment:
```suggestion
    def cache = new HashMap<IdRef>()
```

Reply:
Took the opportunity to declare cache in a groovy fashion and spare the Hashmap import